commcare_connect/utils/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+
+DATE_INPUT = forms.DateInput(format="%Y-%m-%d", attrs={"type": "date"})
+INPUT_CLASS = "base-input"
+TEXTAREA_CLASS = "base-textarea"
+SELECT_CLASS = "base-dropdown"
+CHECKBOX_CLASS = "simple-toggle"
+
+
+FORM_BASE_STYLE = {
+    "textarea": "base-textarea",
+    "select": "base-dropdown",
+    "checkbox": "simple-toggle",
```

Comment:
It looks like except for the checkbox class, we need none of the other classes. The default crispy-tailwind styling works well. In that case, I am not sure if it's worth using this anymore instead of directly using the CHECKBOX_CLASS.

Reply:
Fixed. [f74fd38](https://github.com/dimagi/commcare-connect/pull/585/commits/f74fd38a859506134e96ed495dea7f4ece5f3a47)